models/project/github.go

```diff
@@ -0,0 +1,166 @@
+package project
+
+import (
+	"../../config"
+	"../../modules/hub"
+	"../../modules/logging"
+	"../db"
+	"../list"
+	"../task"
+	"database/sql"
+	"errors"
+
+	"github.com/google/go-github/github"
+)
+
+func (u *ProjectStruct) LoadFromGithub(issues []github.Issue) error {
```

Comment:
__[golint]__
* [golint] exported method ProjectStruct.LoadFromGithub should have comment or be unexported